cmd/tctest/cli/tc.go

```diff
@@ -51,19 +53,33 @@ func TcCmd(server, buildTypeId, branch, testRegEx, user, pass string, wait bool)
 	return nil
 }
 
-func TcBuild(server, buildTypeId, branch, testRegEx, user, pass string, wait bool) (string, string, error) {
+func TcBuild(server, buildTypeId, buildProperties, branch, testRegEx, user, pass string, wait bool) (string, string, error) {
 	url := fmt.Sprintf("https://%s/app/rest/2018.1/buildQueue", server)
+
+	bodyAddtionalProperties := ""
+	if buildProperties != "" {
+		common.Log.Debugf("adding additional properties:")
+		for _, p := range strings.Split(buildProperties, ";") {
+			parts := strings.Split(p, "=")
+			if len(parts) != 2 {
+				return "", "", fmt.Errorf("unable to parse build property '%s': missing =s", p)
+			}
+			common.Log.Debugf("  property:%s=%s", parts[0], parts[1])
```

Comment:
minor: do we want the extra space there?

Reply:
yep yep! aligned under the above debug heading